### PR TITLE
Fix: log_server and client names

### PR DESCRIPTION
**Pull request by oxedions**
Small fixs in logs roles.

### CODE REVIEW

File: roles/core/log_client/tasks/main.yml [hunk 1]
```diff
@@ -70,12 +70,12 @@
...
     - tcp
     - udp
-  notify: Restart rsyslog services
+  notify: service █ Restart rsyslog service
 
 - name: meta █ Run handler tasks to restart rsyslog services
   meta: flush_handlers
 
-- name: Manage services state
+- name: "service █ Manage {{ j2_role_variables['services_to_start'] | join(' ') }} state"
```
Comment (btravouillon) [inline roles/core/log_client/tasks/main.yml:78]: Variable might be wrong here.

File: roles/core/log_server/readme.rst [hunk 2]
```diff
@@ -11,11 +11,12 @@ Instructions
...
+log_server_port variable.
 
 Changelog
 ^^^^^^^^^
 * 1.0.3: Enable log server port customization with log_server_port. strus38
 * 1.0.1: Fixed bad template. Documentation. johnnykeats <johnny.keats@outlook.com>
 * 1.0.0: Role creation. Benoit Leveugle <benoit.leveugle@gmail.com>
- 
\ No newline at end of file
+
```
Comment (btravouillon) [inline roles/core/log_server/readme.rst:22]: Is the additional blank line expected?

**Review by btravouillon — CHANGES_REQUESTED** [review]

File: roles/core/log_server/readme.rst (hunk 2 above)
Comment (oxedions) [reply roles/core/log_server/readme.rst]: Nop. Fixed

File: roles/core/log_client/tasks/main.yml (hunk 1 above)
Comment (oxedions) [reply roles/core/log_client/tasks/main.yml]: Good catch, I must be tired. Fixed.

File: roles/core/log_server/tasks/main.yml [hunk 3]
```diff
@@ -25,7 +25,7 @@
     - service
     - template
 
-- name: "Add services to firewall's {{ log_server_firewall_zone | default('public') }} zone"
+- name:  "firewalld █ Add services to firewall's {{ log_server_firewall_zone | default('public') }} zone"
```
Comment (btravouillon) [inline roles/core/log_server/tasks/main.yml:28]: yaml: too many spaces after colon (colons)
roles/core/log_server/tasks/main.yml:28

File: roles/core/log_server/tasks/main.yml [hunk 4]
```diff
@@ -39,7 +39,7 @@
   tags:
     - firewall
 
-- name: "Add custom port to firewall's {{ log_server_firewall_zone | default('public') }} zone"
+- name:  "firewalld █ Add custom port to firewall's {{ log_server_firewall_zone | default('public') }} zone"
```
Comment (btravouillon) [inline roles/core/log_server/tasks/main.yml:42]: yaml: too many spaces after colon (colons)
roles/core/log_server/tasks/main.yml:42

File: roles/core/log_server/tasks/main.yml [hunk 5]
```diff
@@ -52,14 +52,14 @@
   tags:
     - firewall
 
-- name: Install packages
+- name:  "package █ Install {{ log_server_packages_to_install }} packages"
```
Comment (btravouillon) [inline roles/core/log_server/tasks/main.yml:55]: yaml: too many spaces after colon (colons)
roles/core/log_server/tasks/main.yml:55

**Review by btravouillon — CHANGES_REQUESTED** [review]
Catch by the CI

**Comment by oxedions** [comment]
Should be fixed

**Review by btravouillon — APPROVED** [review]
Looks good to me. 👍

**Pull request merged**